lib/composer_messages_finder.rb

```diff
@@ -212,6 +212,10 @@ def check_reviving_old_topic
     }
   end
 
+  def self.user_not_seen(usernames)
```

Comment:
Out of context, `user_not_seen` doesn't indicate that we are talking about a long time ago. Can't really use `user_not_seen_recently` either IMO, because the default, 2 years, is not recent. 

So... maybe we can call this `user_not_seen_for_months`? Or `user_not_seen_in_a_while`?

Reply:
`user_not_seen_in_a_while` looks good so I went ahead with that!